### DISCUSSION TITLE
Something about NavigationView

### DISCUSSION THREAD

**Discussion by Dawson924** [amwx/FluentAvalonia#531] [Q&A] [answered]
The NavView works fine through MVVM pattern. One thing that doesn't work is the default Settings item in my NavView. How can I know that it has been clicked? Or if there is a built-in function to archive this?

What i am doing right now is to add a setting item manually. But I can't make it stay at the bottom...

```cs
		<ui:NavigationView x:Name="MainNavigationView"
						   Grid.Row="1"
						   MenuItemsSource="{Binding SidebarButtons}"
						   SelectedItem="{Binding SelectedButton}"
						   Content="{Binding CurrentPage}"
						   PaneDisplayMode="Left"
						   OpenPaneLength="240"
						   IsPaneOpen="True"
						   IsSettingsVisible="True"
						   Margin="3 0 0 0">
			
			<ui:NavigationView.MenuItemTemplate>
				<DataTemplate DataType="{x:Type m:SidebarButton}">
					<ui:NavigationViewItem Content="{Binding Label}" IconSource="{Binding Icon}"/>
				</DataTemplate>
			</ui:NavigationView.MenuItemTemplate>
			
		</ui:NavigationView>
```

**Comment by timunie** [comment, marked as answer]
The sample App has all docs in you need: 
![image](https://github.com/amwx/FluentAvalonia/assets/47110241/fffd352f-92f3-47f5-9065-07c6a506b121)

**Reply by OmerFarukAkyapak** [reply]
How to change PaneToggleButton's default icon in NavigationView ?

**Reply by Dawson924** [reply]
Thanks, problem solved
